lib/environment.js

```diff
@@ -5,21 +5,24 @@ module.exports = {
     require: {
         // builtins required by the libraries exposed for sandbox users
         events: {preferBuiltin: true, glob: true},
+        path: {preferBuiltin: true, glob: true},
         timers: {preferBuiltin: true, glob: true},
         _process: {preferBuiltin: true, glob: true},
         util: {preferBuiltin: true, glob: true},
         buffer: {expose: 'buffer', glob: true},
         'liquid-json': {expose: 'json', glob: true},
-        lodash3: {expose: 'lodash', glob: true},
```

Comment:
this was creating conflict inside postman-collection library. this otherwise does not break anything since legacy-execute.js directly requires lodash3